src/Seq.Input.RabbitMQ/RabbitMQInput.cs

```diff
@@ -44,6 +50,12 @@ public class RabbitMQInput : SeqApp, IPublishJson, IDisposable
             HelpText = "The RabbitMQ queue name to receive events from. The default is `Logs`.")]
         public string RabbitMQQueue { get; set; } = "logs";
 
+        [SeqAppSetting(
+            DisplayName = "Ssl",
```

Comment:
`DisplayName` might be better as `"Require SSL"`?

Reply:
changed